### PR TITLE
Lesson 9 Wiki Notes

### PR DESCRIPTION
**Pull request by gildedgardenia**

### CODE REVIEW

File: Class-9-(Exceptions-&-Build-Tools).md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+   1. How you decided what code to put in ``try`` statement and what code to put in ``catch`` statement.
+   1. Does it include a ``finally`` statement? Why or why not?
+1. Review MSBuild and NuGet. Go over what each one does and why each one matters.
+1. Show how to use debugging tools in Visual Studio to see the *Call Stack*, add breakpoints, and address common problems.
+
+## Small Group Time: Class 9 Studio (TA Notes)
+
+1. Students will be sharing their errors and trying to use debugging tools on ``HelloASPDotNETDemo``. Some things to keep in mind:
+   1. Don't be afraid to share your own stories of bugs and errors in your code! Emphasize that every programmer has to deal with bugs and that it gets a lot easier once you have a strategy in place and the tools 
+   1. Ask students why! Helping students figure out a process by which they can debug their code is the point of the studio.
```
Comment (CarlyLanglois) [inline Class-9-(Exceptions-&-Build-Tools).md]: I don't quite get this line - what are the TAs asking students about?

File: Class-9-(Exceptions-&-Build-Tools).md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Class 9: Exceptions and Build Tools
+
+## In the prep work this week, the students learned
+
+1. What an exception is and how to handle an exception.
```
Comment (CarlyLanglois) [inline Class-9-(Exceptions-&-Build-Tools).md]: Actually, we assume that students are familiar with the concept of an exception - the material is more intended to be a review of the idea and how to use them specifically in the c# language

**Review by CarlyLanglois — APPROVED** [review]

File: Class-9-(Exceptions-&-Build-Tools).md (hunk 2 above)
Comment (gildedgardenia) [reply Class-9-(Exceptions-&-Build-Tools).md]: Rephrased!

File: Class-9-(Exceptions-&-Build-Tools).md (hunk 1 above)
Comment (gildedgardenia) [reply Class-9-(Exceptions-&-Build-Tools).md]: Clarified!

**Pull request merged**